atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -1187,26 +1187,13 @@ private static LockAndTimestampServices createRawLeaderServices(
                 leaderConfig,
                 userAgent);
         LeaderElectionService leader = localPaxosServices.leaderElectionService();
-        LockService localLock = ServiceCreator.instrumentService(
-                metricsManager.getRegistry(),
-                AwaitingLeadershipProxy.newProxyInstance(LockService.class, lock::get, leader),
-                LockService.class);
+        LockService localLock = AwaitingLeadershipProxy.newProxyInstance(LockService.class, lock::get, leader);
 
-        ManagedTimestampService managedTimestampProxy =
+        ManagedTimestampService localManagedTimestamp =
                 AwaitingLeadershipProxy.newProxyInstance(ManagedTimestampService.class, time::get, leader);
 
-        TimestampService localTime = ServiceCreator.instrumentService(
-                metricsManager.getRegistry(),
-                managedTimestampProxy,
-                TimestampService.class);
-
-        TimestampManagementService localManagement = ServiceCreator.instrumentService(
-                metricsManager.getRegistry(),
-                managedTimestampProxy,
-                TimestampManagementService.class);
         env.accept(localLock);
-        env.accept(localTime);
-        env.accept(localManagement);
+        env.accept(localManagedTimestamp);
```

Comment:
I'll take this from here. We can't really do this because the JAX-RS semantics for annotation inheritance are different.

Reply:
Ah that explains it, thanks!